.travis.yml

```diff
@@ -1,5 +1,5 @@
 # It may be tempting to add parens around each individual clause in this expression, but Travis then builds pushes anyway
-if: branch = master OR branch =~ ^release/ OR tag IS present
+if: branch = master OR branch =~ ^features/ OR branch =~ ^release/ OR tag IS present
```

Comment:
Intended to check this in?

Reply:
Yes.  It's a standard pattern we use in our repos to make it possible to publish built bits that aren't in master.  it's how ew can test a potential fix cross repo.  We hadn't needed to do that here until now.